### DISCUSSION TITLE
New pricing model - what happens to running services if we exceed credits?

### DISCUSSION THREAD

**Discussion by tchanxx** [serverless/serverless#12376] [Q&A] [answered]
Hi! I currently have four services on Serverless. With the recent pricing update, the free tier only allows for two services. https://www.serverless.com/pricing

What happens to the running services if we exceed the credit allocation? Do they just stop metric recording or are deployments prevented? Thanks.

**Comment by mareksuscak** [comment]
I'd also like to know. Been trying to find some details about the free tier change because our production services went down today and we think it was triggered by us being 5 credits over the limit on serverless.com because upgrading to a paid account and redeploying all services helped.

**Reply by garethmcc** [reply]
I would recommend investigating further. The only changes that occur are UI changes on the Dashboard. All the features and functionality, including capturing monitoring data continues even when credit limit is exceeded so there is nothing that can affect your deployed applications based on being over the purchased credits

**Comment by garethmcc** [comment, marked as answer]
Hi folks. Nothing happens to already deployed services. Nothing gets turned off or deactivated or disabled, just some features of Dashboard access is restricted until credits are added to the subscription.
